streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -1559,10 +1567,18 @@ public Sink(final String name,
 
         @Override
         public String topic() {
-            if (topicNameExtractor instanceof StaticTopicNameExtractor)
+            if (topicNameExtractor instanceof StaticTopicNameExtractor) {
                 return ((StaticTopicNameExtractor) topicNameExtractor).topicName;
-            else
+            }
+            return null;
```

Comment:
I'm ready to admit this might be a personal preference, but I think these are cleaner with both `if` and `else` instead of asymmetric `if` followed by a `return`.